CHANGELOG.md

```diff
@@ -2,12 +2,25 @@
 
 ## Unreleased
```

Comment:
0.5.0 or 0.4.19? Moving `before_send()` to the end of the pre-processing (in #729) could be considered a breaking change.